specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3090 @@
+        }
+      }
+    },
+    "AnalyzeTextEntityLinkingInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/EntityLinkingTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "EntityLinking"
+    },
+    "AnalyzeTextEntityRecognitionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/EntitiesTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "EntityRecognition"
+    },
+    "AnalyzeTextKeyPhraseExtractionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/PreBuiltTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "KeyPhraseExtraction"
+    },
+    "AnalyzeTextPiiEntitiesRecognitionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/PiiTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "PiiEntityRecognition"
+    },
+    "AnalyzeTextLanguageDetectionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/LanguageDetectionAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/LanguageDetectionTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "LanguageDetection"
+    },
+    "AnalyzeTextSentimentAnalysisInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/SentimentAnalysisTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "SentimentAnalysis"
+    },
+    "AnalyzeTextJobsInput": {
+      "properties": {
+        "displayName": {
+          "description": "Optional display name for the analysis job.",
+          "type": "string"
+        },
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
```

Comment:
kind of related to my other comment - should this allow for LanguageDetectionAnalysisInput as well, or do I use the LanguageDetectionTaskInput object to specify country/region hint?

Reply:
No, in the case of the jobs endpoint we won't directly support Language Detection as an explicit task type.  Instead we will (eventually) use automatic language detection for this endpoint.  This feature was originally planned for this preview but will be moved to another preview due to resource constraints (discussed in another comment).